### PR TITLE
Regression with functional response

### PR DESCRIPTION
**Pull request by manso92**

### CODE REVIEW

File: skfda/ml/regression/linear_model.py [hunk 1]
```diff
@@ -93,3 +93,112 @@ def _argcheck(self, y, x, weights = None):
...
+                .to_basis(self.ybasis_)
+                for j in range(X[0].nsamples)]
+
+
+    def _argcheck(self, y, x, weights = None):
+        """Do some checks to types and shapes"""
+        if not isinstance(y, FData):
+            raise ValueError("The explanined variable is not an FData objetc")
+
+        # TODO check for same domain_range
```
Comment (vnmabus) [inline skfda/ml/regression/linear_model.py]: ?

**Comment by manso92** [comment]
Currently developing

**Comment by vnmabus** [comment]
Are you currently working in this PR, @manso92 ?

**Comment by manso92** [comment]
I've fixed the code, I'll rewrite the tests and I'll push it to review it

**Pull request closed without merging**